### PR TITLE
[CHI-370] Add Sidenav collapsed

### PR DESCRIPTION
**Pull request by jllr**
@mattnickles 

https://ctl.atlassian.net/browse/CHI-370

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/254/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/255/ for more details.

File: src/chi/components/sidebar/sidenav.scss [hunk 1]
```diff
@@ -197,10 +198,56 @@ $sidenav-xl-width: 12.5rem;
...
+
+  &.-collapsed {
+    width: $sidenav-collapsed-width;
+
+    .m-sidenav__list > li {
+      > a {
+        font-size: .8125rem;
+        font-weight: 400;
+        min-height: 4rem;
+
```
Comment (mattnickles) [inline src/chi/components/sidebar/sidenav.scss]: This doesn't support multi-line labels. Need to center the text and set a line-height on the span.

File: src/chi/components/sidebar/sidenav.scss [hunk 2]
```diff
@@ -197,10 +198,56 @@ $sidenav-xl-width: 12.5rem;
       }
     }
   }
+
+  &.-collapsed {
+    width: $sidenav-collapsed-width;
+
+    .m-sidenav__list > li {
+      > a {
+        font-size: .8125rem;
```
Comment (mattnickles) [inline src/chi/components/sidebar/sidenav.scss]: Shouldn't these styles be defined on the small viewport/responsive version of base so the two aren't inconsistent?

File: src/chi/components/sidebar/sidenav.scss [hunk 3]
```diff
@@ -197,10 +198,56 @@ $sidenav-xl-width: 12.5rem;
       }
     }
   }
+
+  &.-collapsed {
```
Comment (mattnickles) [inline src/chi/components/sidebar/sidenav.scss]: Not sure about this name. The base menu has two responsive collapsed states

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/257/ for more details.

File: src/chi/components/sidebar/sidenav.scss (hunk 3 above)
Comment (mattnickles) [reply src/chi/components/sidebar/sidenav.scss]: @jllr Let's use size classes -xs -sm and -md

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/264/ for more details.

**Pull request merged**